internal/utils/utils.go

```diff
@@ -56,3 +57,13 @@ func GetTimeDifferenceInDaysFrom(t time.Time, s time.Time) (diff int) {
 	dur := s.Sub(t)
 	return int(math.Floor(dur.Hours() / 24))
 }
+
+// GetTimeDifferenceInWeeks Gets the time difference in weeks between time arg and current time
+func GetTimeDifferenceInWeeks(t time.Time) (diff int) {
+	return GetTimeDifferenceInDays(t) / 7
+}
+
+// GetCurrentSeasonWeek Gets the current week number of the season
+func GetCurrentSeasonWeek() (week int) {
```

Comment:
Do we need this? My motivation behind keeping the start dates in each individual file was to make each file the single source of truth for that activity; so if you wanted Nightfall rotation data, all the data related to that would be in `nightfalls.json`. The dates do repeat but I would actually do it the way you did if we didn't have to update the data files every season. Unfortunately, you need to _manually_ update them for the first few weeks of the season.. they don't put the rotation on the API either so /shrug

Do you think we could remove `GetCurrentSeasonWeek` and `season_data`? This would mean you'd have to add `start_date` to the `dungeons.json` file and use `ReadRotationData` & `GetTimeDifferenceInDays` (or your new `GetTimeDifferenceInWeeks` :))

Reply:
I added the season data (in `season.json`) as a shot-in-the-dark to see your appetite for having season-specific data (start date, name, whatever else). I will remove it

Reply:
In my head, once the bot gets to that point where we start serving seasonal data beyond dungeons, nightfalls, lost sectors.. whatever.. we should be using the Bungie API in which case we'd probably need a database and move away from these JSON files.. but that's a problem for another day. I think for now we can skip this, yeah but can always add it back in when we need it!